demos/pedestrian_tracker_demo/main.cpp

```diff
@@ -50,7 +50,11 @@ CreatePedestrianTracker(const std::string& reid_model,
 
     if (!reid_model.empty() && !reid_weights.empty()) {
         CnnConfig reid_config(reid_model, reid_weights);
-        reid_config.max_batch_size = 16;
+        reid_config.max_batch_size = 16;   // defaulting to 16
+        if ((deviceName.find("MYRIAD") != std::string::npos) || (deviceName.find("HDDL") != std::string::npos)) {
```

Comment:
I think here you should use that approach with `GetConfig`, it just needs to add catching of `InferenceEngine::details::InferenceEngineException`.

Reply:
Changed to try.. catch..

Reply:
This block of code is misaligned now. The rest looks fine.